recipes/pyddx/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "pyddx" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "d5cde4473c19b836a77948f79e33ec15b8040fdbf2278a8f02d7c81df4f09111" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ddsolvation/ddx/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

No need for the indirection here.
```suggestion
  sha256: d5cde4473c19b836a77948f79e33ec15b8040fdbf2278a8f02d7c81df4f09111
```